backend/src/akvo/lumen/auth.clj

```diff
@@ -129,13 +129,13 @@
       (if (api-authz? jwt-claims)
         (try
           (let [email (get jwt-claims "email")
-                allowed-paths (delay (keycloak/allowed-paths keycloak email))]
+                allowed-paths (keycloak/allowed-paths keycloak email)]
```

Comment:
We want a `delay`. See https://github.com/akvo/akvo-lumen/pull/2227/commits/3b2fc2e59d2993215679fa711cde601d6f440d14#r296692858